### DISCUSSION TITLE
What to include in the initial version of the website?

### DISCUSSION THREAD

**Discussion by salmonb** [webfx-project/webfx-website#10] [General]
The purpose of this discussion is to define the minimal version of the website that will be good enough to publish.

For example, we already started talking about a new card, but can it be added later? Should we make a first version of the website without it?

In other words, what do you think are the minimal steps required to transform the current website preview into a first publishable version?

**Comment by dukke** [comment]
The demos you already have on the site are awesome and run really fast! I think just showing them would probably be enough to attract attention?

**Reply by salmonb** [reply]
Yes I also think the demos are ok. But the question was more about the cards. Or are you suggesting to make a first version of the website with only the demos?
